README.md

```diff
@@ -173,10 +173,10 @@ And invoked via `appInsights.<METHOD_NAME>`
 | ----------------------------|------------------------------------------------------------------------------------------------------------|
 | setDistributedTracingMode   | AI only tracing mode is no longer supported. Migrate to using W3C_AND_AI tracing mode. |
 | setAutoCollectHeartbeat     | Heartbeat is not supported in either the distro or the shim.|
-| enableWebInstrumenatation   | WebInstrumentation is not supported in either the distro or the shim. |
+| enableWebInstrumenatation   | WebInstrumentation is not supported in the shim, but is available in the distro as `enableBrowserSdkLoader` |
 | setAutoDependencyCorrelation| Turning off autoDependencyCorrelation is not supported by either the distro or the shim. |
 | setUseDiskRetryCaching      | While enabling/disabling offline storage is supported, setting the resend interval or the maxBytesOnDisk values are not supported in the shim or the distro. |
-| setAutoCollectIncomingRequestAzureFunctions | Auto collection of Azure Functions is not supported by the shim or the distro. |
+| setAutoCollectIncomingRequestAzureFunctions | Not supported by the shim. Migrate to the distro to use auto collection of Azure Functions. |
```

Comment:
Auto collection of incoming requests is not supported in distro either

Reply:
Corrected and addressed the contradiction in the JSON config table.